scss/bitstyles/generic/_normalize.scss

```diff
@@ -170,14 +170,6 @@ button:-moz-focusring,
   outline: 1px dotted ButtonText;
 }
 
-/**
- * Correct the padding in Firefox.
- */
-
-fieldset {
-  padding: size.get('m');
```

Comment:
drive-by?

Reply:
Yes kinda :) It’s related to the issue, in that it’s about default spacing applied to elements at this early stage, which is why I found it. This property gets re-specified later in the forms file (using a variable that can be overridden in projects, which for typography was the purpose of the PR), so this line was not needed